src/components/Button/index.js

```diff
@@ -0,0 +1,23 @@
+import { h, Component } from 'preact';
+import { Link } from 'preact-router/match';
+import classNames from 'classnames';
+
+import style from './style.scss';
+
+export default ({
+  type = 'primary',
```

Comment:
Wouldn't it be better to pass simple 'secondary' or 'primary' instead of type prop? There is no need to use BEM here, because of css-modules :)
we can add in classnames class .primary or .secondary as a boolean so our scss will look like:
```
.button {
  & .primary {...}
}

Reply:
Actually that's good news :) I don't really like BEM so I'll refactor it later and also got another idea to fix that class names